.github/workflows/integrate.yaml

```
@@ -46,17 +46,22 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        charm: [controller]
+        charm: [controller, ui]
     steps:
     - name: Check out code
       uses: actions/checkout@v3
     - name: Setup operator environment
-      uses: charmed-kubernetes/actions-operator@main
+      # TODO: Unpin this when https://github.com/charmed-kubernetes/actions-operator/pull/46 is merged
+      uses: claudiubelu/actions-operator@18ebf92ae3043bd3dd15238e5d9b662d7ba08daf
```

[this release](https://github.com/charmed-kubernetes/actions-operator/releases/tag/1.0.0) should work here.  It added the `microk8s-addons` argument
```suggestion
      # TODO: Unpin this when https://github.com/charmed-kubernetes/actions-operator/pull/46 is merged
      uses: charmed-kubernetes/actions-operator@1.0.0
```

---

This same change can be done [here](https://github.com/canonical/notebook-operators/pull/97/files#diff-d684acc09c646fd347d0ee93598101fb403664cc70e38cb6f671355ab6de1854R87) as well